source/class/qx/tool/compiler/Es6ify.js

```diff
@@ -168,7 +168,14 @@ qx.Class.define("qx.tool.compiler.Es6ify", {
         passPerPreset: true
       };
 
-      let result = babelCore.transform(src, config);
+      let result;
+      while (true) {
```

Comment:
```suggestion
      let cycleCount = 0;
      while (true) {
```

Reply:
lol. I'll assume you pressed the green button before you were finished typing in your suggestion...